sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -345,6 +418,56 @@ internal static IReadOnlyList<EntityRecognitionTasksItem> ConvertToEntityRecogni
             return collection;
         }
 
+        internal static IReadOnlyCollection<ExtractKeyPhrasesActionResult> ConvertToExtractKeyPhrasesActionResults(AnalyzeJobState jobState, IDictionary<string, int> idToIndexMap)
+        {
+            var collection = new List<ExtractKeyPhrasesActionResult>();
+
+            // TODO - Map Error.Target to ActionResult
```

Comment:
so is the idea to not implement this for this preview? not sure I follow

Reply:
updated